carols/riu.ly

```diff
@@ -0,0 +1,465 @@
+\version "2.18.0"
```

Comment:
Was this intended to be included in this PR? It's supposed to go with the one that adds it right?

Reply:
lol idk man